src/napari_imagej/_function.py

```diff
@@ -399,13 +529,15 @@ def __init__(self, napari_viewer: Viewer):
 
         self.layout().addWidget(self._search_widget)
 
-        self.searcher: QLineEdit = self._generate_searcher()
+        self.searchers, self.resultConverters = self._generate_searchers()
         self.searchService = self._generate_search_service()
 
         # Results box
-        self.results = []  # type: ignore
-        self.tableWidget: QTableWidget = self._generate_results_widget()
-        self.layout().addWidget(self.tableWidget)
+        self.results = [[] for _ in range(len(self.searchers))]
```

Comment:
Why are the results an array of arrays?

Because there are modules *and* ops, I know, but we should describe that.